README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
 
 1. `aws organizations create-account --email "username+cli-test@redhat.com" --account-name "username-cli-test" --profile=orgtest`
-
 2. `aws organizations list-accounts --profile=orgtest | jq '.[][] | select(.Name=="username-cli-test")'`
+3. `aws sts assume-role --role-arn arn:aws:iam::<ID>:role/OrganizationAccountAccessRole --role-session-name username-cli-test --profile=orgtest`
+
+# 2. The Custom Resources
+
+## 2.1. AccountPool CR 
+
+The AccountPool CR holds the information about the available number of accounts that can be claimed for cluster provisioning.
+
+```yaml
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: AccountPool
+metadata:
+  name: example-accountpool
+  namespace: aws-account-operator
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/aws-account-operator/accountpools/example-accountpool
+  uid: 9979786a-a8cb-11e9-a2a3-2a2ae2dbcce4
+spec:
+  poolSize: 50
+status:
+  claimedAccounts: 1946
+  poolSize: 50
+  unclaimedAccounts: 50
+```
+
+## 2.2. Account CR
+
+The Account CR holds the details about the AWS account that was created, where the account is in the process of becoming ready, and whether its linked to an AccountClaime, i.e. claimed. 
+
+```yaml
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: Account
+metadata:
+  creationTimestamp: 2019-07-03T16:07:16Z
+  finalizers:
+  - finalizer.aws.managed.openshift.io
+  generation: 4
+  name: osd-{accountName}
+  namespace: aws-account-operator
+  ownerReferences:
+  - apiVersion: aws.managed.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: AccountPool
+    name: example-accountpool
+    uid: 9979786a-a8cb-11e9-a2a3-2a2ae2dbcce4
+  resourceVersion: "49984188"
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/aws-account-operator/accounts/osd-{accountName}
+  uid: a07a6867-9dac-11e9-b4bb-0e6ed767b7c0
+spec:
+  awsAccountID: "0000000000"
+  claimLink: example-link
+  iamUserSecret: osd-{accountName}-secret
+status:
+  claimed: true
+  conditions:
+  - lastProbeTime: 2019-07-03T16:14:50Z
+    lastTransitioNTime: 2019-07-03T16:14:50Z
+  	message: Attempting to create account
+ 	  reason: Creating
+ 	  status: "True"
+ 	  type: Creating
+  - lastProbeTime: 2019-07-03T16:18:55Z
+    lastTransitioNTime: 2019-07-03T16:18:55Z
+    message: Account pending AWS limits verification
+    reason: PendingVerification
+    status: "True"
+    type: PendingVerification
+  - lastProbeTime: 2019-07-05T13:19:32Z
+    lastTransitioNTime: 2019-07-05T13:19:32Z
+    message: Account ready to be claimed
+    reason: Ready
+    status: "True"
+    type: Ready
+  state: Ready
+  supportCaseID: case-000000000-muen-2019-000000000000
```

Comment:
Same here, if we want to show the status I think we should call it out in a separate section discussing the accounts status since its important.